### PR TITLE
Before & Between queries for Dates

### PR DESCRIPTION
**Pull request by andyklimczak**
Ability to filter with a before date, or between two dates

Love your library, thanks !

### CODE REVIEW

**Comment by Starefossen** [comment]
Thanks a lot @andyklimczak 😄 Are you interested in becoming a collaborator to this library?

**Comment by andyklimczak** [comment]
rebased and changed commit message, should be good now

and I would love to help !

File: index.js [hunk 1]
```diff
@@ -103,13 +112,43 @@ module.exports.prototype.customAfter = field => (query, value) => {
...
+
+  if (date.toString() !== 'Invalid Date') {
+    query[field] = {
+      $lt: date.toISOString(),
+    };
+  }
+};
+
+module.exports.prototype.customBetween = field => (query, value) => {
+  const [afterValue, beforeValue] = value.split('|');
```
Comment (Starefossen) [inline index.js]: At the moment we need to support Node.js 4. Destructuring assignments are only supported in newer versions of Node.js (from version 6 and forward).

```js
const dates = value.split('|');
const afterValue = dates[0];
const beforeValue = dates[1];
```

Comment (andyklimczak) [inline index.js]: makes sense 👍

**Comment by Starefossen** [comment]
Great work @andyklimczak 🎉

**Pull request merged**